src/components/Tasks/TaskStatus.js

```diff
@@ -1,7 +1,37 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { textStyle, GU } from '@aragon/ui'
+import dayjs from 'dayjs'
```

Comment:
Wondering, should we make sure to always import `dayjs` from `lib/dayjs`? I assume only that import has the extended functionality?

Reply:
yes totally